src/views/hotspot.handlebars

```diff
@@ -8,8 +8,8 @@
      <VersionHigh>2.0</VersionHigh>
      <VersionLow>1.0</VersionLow>
      <AccessProcedure>1.0</AccessProcedure>
-     <AccessLocation>Mozilla IoT Gateway</AccessLocation>
-     <LocationName>Mozilla IoT Gateway</LocationName>
+     <AccessLocation>{{ap_ssid}}</AccessLocation>
+     <LocationName>{{ap_ssid}}</LocationName>
      <LoginURL>http://{{ap_ip}}/wifi-setup</LoginURL>
```

Comment:
Ugh - I don't think it should just be the Mac address. I think it should be something like Mozilla IoT Gateway followed by the last 6 digits of the Mac address.

Reply:
It is. It will end up being `Mozilla IoT Gateway 9BC2`, for example.

Reply:
Nevermind - i totally misread this

Reply:
@mrstegeman  Sorry for the late reply.

I agree with last four digits. Sorry to be a pain but can we make this e.g. "Things Gateway 9BC2" as in the mockups rather than "Mozilla IoT Gateway 9BC2". It was never meant to be "Mozilla IoT Gateway" in the first place because Mozilla IoT is the name of the team, not the software. I realise Mozilla IoT has the benefit that it has Mozilla's brand name in it, but I anticipate changing default again when we do our re-brand.

Reply:
Oh, you may already have done this in #1686

Reply:
Yep, done in the other PR.